src/auth/dto/index.ts

```diff
@@ -6,3 +6,7 @@ export * from './user-request';
 export * from './create-pat-response';
 export * from './access-token-payload';
 export * from './change-password-request';
+export * from './create-qrcode-2fa-response';
+export * from './verify-2fa-request';
+export * from './recovery-2fa-request';
+export * from './enable-2fa-response';
```

Comment:
if you have a vscode for organizing imports then use it to have imports in alphabetical order 🙂

Reply:
![Capture-2023-09-13-112256](https://github.com/KubaBaniak/RecipeAppNestjs/assets/91290924/ba4d7854-1934-4cee-a092-2d4bcb22fbcf)

Reply:
I will find the reason why my editor doesn't apply auto-sorting and fix it in my free time 😉